### ISSUE TITLE
AssertionError in visualzatiion.format_word_importances()

### ISSUE THREAD

**Issue by RylanSchaeffer** [pytorch/captum#785]
I'm getting an `AssertionError` from https://github.com/pytorch/captum/blob/master/captum/attr/_utils/visualization.py#L500 and I'm not sure sure why.

The sentence is `'explorer: morseth have whiskers\nstudent: what color?\nexplorer: they can be any color, as long as they have whiskers\nstudent: do they have fur?\nexplorer: no\nstudent: do they live in the water?\nexplorer: yes'`

When I pass the sentence through `DistilBertPretrainedTokenizer`, I get a length 53 `input_ids`:

```
    tokenized_sentences = tokenizer(
        [sentence],
        truncation=False,
        padding=False)
```

The tokenized sentence is:

```
[101, 10566, 1024, 17107, 2705, 2031, 1059, 24158, 11451, 3076, 1024, 2054, 3609, 1029, 10566, 1024, 2027, 2064, 2022, 2151, 3609, 1010, 2004, 2146, 2004, 2027, 2031, 1059, 24158, 11451, 3076, 1024, 2079, 2027, 2031, 6519, 1029, 10566, 1024, 2053, 3076, 1024, 2079, 2027, 2444, 1999, 1996, 2300, 1029, 10566, 1024, 2748, 102]
```

Whereas inside the `format_word_importances()` call, the length of the sentence is 205.

**Comment by NarineK** [comment]
@RylanSchaeffer, is this still an outstanding issue ? It looks like  The importance scores and input features had different lengths ?

**Comment by NarineK** [comment]
Addressed in #804
